.github/workflows/update_submodule.yml

```diff
@@ -0,0 +1,40 @@
+name: Get latest version of submodules and push back to 6-dev branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout
+      - uses: actions/checkout@v3
+        with: 
+          ref: 6-dev
+      
+      # Pull latest plone.api
+      - name: Pull latest plone.api
+        run: |
+          cd submodules/plone.api
+          git pull
+
+      # Pull latest plone.restapi
+      - name: Pull latest plone.restapi
+        run: |
+          cd submodules/plone.restapi
+          git pull
+
+      # Pull latest Volto
+      - name: Pull latest Volto
+        run: |
+          cd submodules/volto
+          git pull
+
+      # Update, commit, and push
+      - name : Update tips
+        run: |
+          git status
+          git add submodules/plone.api submodules/plone.restapi submodules/volto
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git diff --exit-code && git commit -m "Update submodules tip" && git push
```

Comment:
I assume that this workflow only fires when there are diffs in the `docs/src/` directories of each submodule. For example, in Volto, Netlify has this configuration.

https://github.com/plone/volto/blob/5e6fab072500364dd239d44fa6bec76e627bb749/netlify.toml#L4-L5